active/NumberBox/NumberBox.md

```diff
@@ -173,8 +173,4 @@ For example, implementation details. -->
 
 ## Open Questions
 
-* Should we support some basic format types for numerical strings or only mathematics and allow custom format strings? Need to rationalize this with InputScopeNameValue and CoreTextInputScope.
-* Custom format property or traditional format string? 
-* Prefix/Suffix excluded to be TextBox improvements instead? How will prefix and suffix values be included in localization and automation output?
-* Embedded calculator? 
-* Tentative APIs for decimal precision and rounding?
+* For touch/virtual keyboards, can we intelligently swap to the numerical input keyboard?
```

Comment:
I'm not sure I understand this question. Do you mean, 'Can the InputScope always be set to "Number?"'

Reply:
@robloo brought up the idea of visually limiting on-screen keyboard to numerical input only when a NumberBox is the target of focus for the clarity and convenience of end users. I will need to familiarize myself further with input scoping/on-screen keyboard to see if this functionality is something that on-screen keyboard is already designed to support.

Reply:
It would be nice if the On-Screen Keyboard could include a "Calculator" layout that could be useful.  But that is not in the control of WinUI 3.0 ...

https://docs.microsoft.com/en-gb/windows/uwp/design/input/use-input-scope-to-change-the-touch-keyboard

Number seems to be the only option that makes sense

Reply:
Currently this kind of configuration for on-screen keyboard is not available as this keyboard configuration is redundant to calculator scenarios (which show this scope limited input display on-screen already. Closing for now but can revisit for V2.